### ISSUE TITLE
2-stage translation with Ssnpm

### ISSUE THREAD

**Issue by jiahzhang** [riscv/riscv-j-extension#66]
In section 3.5, the spec says
```
When running with virtualization in VS/VU mode with vsatp.MODE = Bare, this means that those two bits may be subject to pointer masking, depending on hgatp.MODE and senvcfg.PMM.
```

Should this not say "either henvcfg.PMM (for VS) or senvcfg.PMM (for VU)"?

**Comment by martinmaas** [comment]
Yes, that's correct – I will add clarification to the text. I will leave this issue open until then.

**Comment by martinmaas** [comment]
This should be addressed as of 567bbd0.
